plugins/connectrpc/go/v1.16.2/buf.plugin.yaml

```diff
@@ -0,0 +1,20 @@
+version: v1
+name: buf.build/connectrpc/go
+plugin_version: v1.16.2
+source_url: https://github.com/connectrpc/connect-go
+integration_guide_url: https://connectrpc.com/docs/go/getting-started
+description: Generates client and server stubs for connectrpc.com/connect. Compatible with the gRPC, gRPC-Web, and Connect RPC protocols.
+deps:
+  - plugin: buf.build/protocolbuffers/go:v1.33.0
```

Comment:
Can we bump this?

Reply:
We can update - looks like I was working on an earlier version that didn't include this plugin.